lib/generators/mountain_view/extra_pages_generator.rb

```diff
@@ -0,0 +1,35 @@
+require_relative './mountain_view_generator_base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.